test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -391,28 +395,24 @@ def match_msg(self, lines, msg, allmatch=False, regexp=False,
                         specified time
         :param syslog: If True, checks for syslog format of date.
                        Defaults to False.
-        :type bool
+        :type syslog: bool
         """
         linecount = 0
         ret = []
         if lines:
             for l in lines:
                 if syslog:
-                    if starttime is not None:
-                        starttime = int(starttime)
-                    if endtime is not None:
-                        endtime = int(endtime)
-                    dt_str = self.get_syslog_date_str(logline=l)
+                    dt_str = self.convert_syslog_date_str(logline=l)
```

Comment:
Use a different variable than 'l'.  It can be confusing.  How about just 'line' or 'tl' or something other than just 'l'.